docs/topics/testing/overview.txt

```diff
@@ -257,6 +257,14 @@ To prevent serialized data from being loaded twice, setting
 :data:`~django.db.models.signals.post_migrate` signal when flushing the test
 database.
 
+.. versionchanged:: 1.11
```

Comment:
Please make the note "self-contained" as described at https://docs.djangoproject.com/en/dev/internals/contributing/writing-documentation/#documenting-new-features.